assets/js/script.js

```diff
@@ -0,0 +1,119 @@
+var pokemonContainerEl = document.querySelector(".poke-card");
+var currentPokemonEl = document.querySelector(".poke-name");
+var currentTypeEl = document.querySelector(".poke-type");
+var currentImageEl = document.querySelector(".poke-image");
+var currentMoveEl = document.querySelector(".poke-move");
+//var currentHabitatEl = document.querySelector("#pokemon-habitat");
+//var apiURL = "https://pokeapi.co/api/v2/pokemon/"
+
+var getPokemon = function() {
+    //format the PokeAPI data 
+    var apiURL = "https://pokeapi.co/api/v2/pokemon/bulbasaur/"; 
+    //make a request to the URL(404 ERROR and network connectivity)
+    fetch(apiURL).then(function(response) {
+        //request for data was successful 
+        if (response.ok) { //"ok" - when the HTTP request status code is something in the 200s - ok = true 404 error
+            response.json().then(function(pokemon) {
+                    console.log(pokemon);
+                    displayPokemon(pokemon);
+                    pokemonImage(pokemon);    
+            });
+        } else { //ok = false (not in the 200s)
+            alert("Error: " + response.statusText); //statusText property - what the issue is 
+        }
+    })
+};
+
+//display Pokemon name, type and moves 
+var displayPokemon = function(pokemon) {
+    var pokeOne = pokemon.name;
+    var pokeOneType = pokemon.types[0].type.name;
+    var pokeTwoType = pokemon.types[1].type.name; 
+    // if (pokeOneType === "grass") {
+    //     window.alert("Hi!");
+    // }
+    
+    currentPokemonEl.textContent = pokeOne;
+
+    currentTypeEl.textContent = pokeOneType + " / " + pokeTwoType; 
+
+    for(var i=0; i<3; i++) {
+        // var pokeMove = pokemon.moves[i].move.name;
+        // console.log (pokeMove);
+        var moveOne = pokemon.moves[Math.floor(Math.random() * 10)].move.name; 
+        var moveTwo = pokemon.moves[Math.floor(Math.random() * 10)].move.name; 
+        var moveThree = pokemon.moves[Math.floor(Math.random() * 10)].move.name; 
+        currentMoveEl.textContent= moveOne + " / " + moveTwo + " / " + moveThree;
+    }  
+}
+
+//display Pokemon image
+var pokemonImage = function (pokemon) {
+    var pokeNumber = pokemon.id; 
+    console.log(pokemon.id);
+    var pokeImage = document.createElement("img");
+    pokeImage.setAttribute("style", "width:250px;height:250px;")
+
+    pokeImage.srcset = `https://pokeres.bastionbot.org/images/pokemon/` + pokeNumber + `.png`;
```

Comment:
@dvicj If you're using backticks, I think it's better to use it this way:
`https://https://pokeres.bastionbot.org/images/pokemon/${pokeNumber}.png`